lib/xgu/xgux.h

```diff
@@ -3,13 +3,71 @@
 
 #include "xgu.h"
 
+#define MIN(a,b) ((a)<(b)?(a):(b))
+#define MAX_BATCH 120
+
 inline
 void xgux_draw_arrays(XguPrimitiveType mode, unsigned int start, unsigned int count) {
     uint32_t *p = pb_begin();
     p = xgu_begin(p, mode);
     p = xgu_draw_arrays(p, start, count);
     p = xgu_end(p);
-    p = pb_end(p);
+    pb_end(p);
+}
+
+inline
+void xgux_draw_elements16(XguPrimitiveType mode, unsigned int count, const uint16_t* elements) {
```

Comment:
API is a bit confusing because this is `xgux_draw_elements..(..., count elements)` vs `xgu_element..(..., elements, count)`.

Note difference of *element* vs *elements* + order of arguments (XGU = closer to `memcpy` / XGUX closer to `glDrawElements`).

Needs a comment in XGU branch after merge to remind us.